sklearn/svm/classes.py

```diff
@@ -1155,6 +1155,14 @@ class OneClassSVM(BaseLibSVM, OutlierMixin):
         The offset is the opposite of `intercept_` and is provided for
         consistency with other outlier detection algorithms.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import make_circles
+    >>> from sklearn.svm import OneClassSVM
+    >>> X, y = make_circles(n_samples=6, factor=0.4, random_state=41)
```

Comment:
Maybe it'd make more sense to have a more realistic dataset for anomality detection, e.g. make_blobs + some random samples as in https://scikit-learn.org/stable/auto_examples/plot_anomaly_comparison.html#sphx-glr-auto-examples-plot-anomaly-comparison-py

I'm OK with it anyway so LGTM, but not pressing merge for someone to double check.

Reply:
thank you @NicolasHug